### PR TITLE
Change plotting format of SW example

### PR DESCRIPTION
**Pull request by maojrs**
I changed the plotting style to match the other plots in the `FV_compare` notebook. It shows now both the depth and the momentum, and it uses the same format and colors as the examples below it.

### CODE REVIEW

**Comment by rjleveque** [comment]
Thanks @maojrs, I like it.

For some reason after I execute the first euler example and then go back up and move the slider bar for the SWE example that code crashes with an error
```
ValueError: x and y must have same first dimension, but have shapes (80,) and (40,)
```
and plots where it looks like the true solution from Euler is being plotted by the SWE widget.  If I re-execute the SWE cells then the slider works fine again but then moving the Euler slider down below crashes that widget.  I can't figure out what's going on with it, maybe you or @ketch will have an idea.

File: FV_compare.ipynb [hunk 1]
```diff
@@ -129,31 +129,42 @@
    "outputs": [],
    "source": [
     "N = 40  # number of grid cells to use\n",
+    "q_l = [10,0]\n",
+    "q_r = [0.5,0]\n",
     "\n",
+    "# Roe soluction\n",
```
Comment (ketch) [inline FV_compare.ipynb]: solution

**Comment by ketch** [comment]
The crash is because `xc_st` is being redefined.  It can be fixed by using a different name (it's always very dangerous to reuse variable names that are related to widgets).

**Comment by maojrs** [comment]
My bad, it should be fixed now. I also renamed the exact solution in the Weno example. It is the same solution as in the previous example, but if the user plays with the initial values, it might have had similar issues. I tried to keep the names consistent with the examples.

**Comment by rjleveque** [comment]
Thanks, I figured it was something like that.

**Pull request merged**